sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosDiagnosticsContext.java

```diff
@@ -375,14 +376,14 @@ public Set<String> getContactedRegionNames() {
      * a custom {@link CosmosDiagnosticsHandler}
      * @return the system usage
      */
-    public CosmosDiagnosticsSystemUsageSnapshot getSystemUsage() {
+    public Map<String, String> getSystemUsage() {
```

Comment:
nit: 
```suggestion
    public Map<String, Object> getSystemUsage() {
```

Object might be a little easier to evolve

Reply:
Changed to Map<String, Object>